src/DocBlock/Tags/See.php

```diff
@@ -56,6 +56,10 @@ public static function create(
         $parts       = preg_split('/\s+/Su', $body, 2);
         $description = isset($parts[1]) ? $descriptionFactory->create($parts[1], $context) : null;
 
+        if (preg_match('{^https?://}', $parts[0])) {
+            return null;
```

Comment:
I'm not against it, but I have no clue what to return, I can't return a See instance as that expects a reference/Fqsen.